CHANGELOG.md

```diff
@@ -1,4 +1,80 @@
+- Update cluster doc [\#117](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/117) ([themantissa](https://github.com/themantissa))
+- Update backup, add links [\#116](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/116) ([themantissa](https://github.com/themantissa))
+- Projects: adding teams attribute [\#113](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/113) ([PacoDw](https://github.com/PacoDw))
+- Update cluster.html.markdown [\#112](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/112) ([themantissa](https://github.com/themantissa))
+- Fix DiskSizeGB missing [\#111](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/111) ([marinsalinas](https://github.com/marinsalinas))
+- Terraform resource for MongoDB Custom Roles [\#108](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/108) ([PacoDw](https://github.com/PacoDw))
+- Fix peering resource [\#107](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/107) ([PacoDw](https://github.com/PacoDw))
+- Fix \#68: Added the ability to re-create the whitelist entry when it's remove manually [\#106](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/106) ([PacoDw](https://github.com/PacoDw))
+- Updating `git clone` command to reference current repository [\#103](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/103) ([macintacos](https://github.com/macintacos))
+- Cluster label and plugin attribute [\#102](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/102) ([PacoDw](https://github.com/PacoDw))
+- Added functions to handle labels attribute in some resources [\#101](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/101) ([PacoDw](https://github.com/PacoDw))
+- Added labels attr for Database User resource [\#100](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/100) ([PacoDw](https://github.com/PacoDw))
+- Alert configuration resource and data source [\#99](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/99) ([PacoDw](https://github.com/PacoDw))
+- Update database\_user.html.markdown [\#98](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/98) ([themantissa](https://github.com/themantissa))
+- update containers and ip whitelist doc [\#96](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/96) ([themantissa](https://github.com/themantissa))
+- Add provider\_name to containers data source [\#95](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/95) ([marinsalinas](https://github.com/marinsalinas))
+- Whitelist [\#94](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/94) ([PacoDw](https://github.com/PacoDw))
```

Comment:
Move to
Implemented enhancements to existing resources:

Reply:
Add - the link is not live now but will be later.
(Breaking Change - Upgrade Guide - https://www.mongodb.com/blog/post/upgrade-guide-for-terraform-mongodb-atlas-040)